src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -524,6 +524,55 @@ public String toString() {
 
   };
 
+  /**
+   * Create an Access Control Log Entry
+   * Result of ACL LOG command
+   */
+  public static final Builder<List<AccessControlLogEntry>> ACCESS_CONTROL_LOG_ENTRY_LIST = new Builder<List<AccessControlLogEntry>>() {
+
+    Map<String,Builder> mappingFunctions = createDecoderMap();
+
+    private Map<String, Builder> createDecoderMap() {
+
+      Map<String,Builder> tempMappingFunctions = new HashMap<>();
+      tempMappingFunctions.put(AccessControlLogEntry.COUNT ,LONG);
+      tempMappingFunctions.put(AccessControlLogEntry.REASON ,STRING);
+      tempMappingFunctions.put(AccessControlLogEntry.CONTEXT ,STRING);
+      tempMappingFunctions.put(AccessControlLogEntry.OBJECT ,STRING);
+      tempMappingFunctions.put(AccessControlLogEntry.USERNAME,STRING);
+      tempMappingFunctions.put(AccessControlLogEntry.AGE_SECONDS,STRING);
+      tempMappingFunctions.put(AccessControlLogEntry.CLIENT_INFO,STRING);
+
+      return  tempMappingFunctions;
+    }
+
+    @Override
+    public List<AccessControlLogEntry> build(Object data) {
+
+      if (null == data) {
+        return null;
+      }
+
+      List<AccessControlLogEntry> list = new ArrayList<>();
+      List<Object> logEntries = (List<Object>)data;
+      Iterator<Object> logsArray = logEntries.iterator();
```

Comment:
```java
List<List<Object>> logEntries = (List<List<Object>>)data; 
for  (List<Object> logEntryData : logEntries) {
```

Reply:
I used the same notation as the `StreamConsumersInfo`, should I open a small issue for improving other part of the code to?

Reply:
Done